debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorTask.java

```diff
@@ -75,6 +75,9 @@ public void start(Configuration config) {
         final TopicSelector<TableId> topicSelector = SqlServerTopicSelector.defaultSelector(connectorConfig);
         final SchemaNameAdjuster schemaNameAdjuster = SchemaNameAdjuster.create(LOGGER);
 
+        // By default do not load whole result sets into memory
```

Comment:
@jpechane Won't this override any value given for these parameters by the user?

Reply:
@gunnarmorling No, that's why I am using withDefault isntead of wit

Reply:
Gotcha. Thx.